### PR TITLE
fix cpu <-> gpu test, hopefully for good

### PR DESCRIPTION
**Pull request by joelgrus**

### CODE REVIEW

File: tests/service/predictors/srl_test.py [hunk 1]
```diff
@@ -46,4 +46,11 @@ def test_cpu_vs_gpu(self):
             prediction_cpu = predictor_cpu.predict_json({"sentence": sentence})
             prediction_gpu = predictor_gpu.predict_json({"sentence": sentence})
 
-            assert prediction_cpu == pytest.approx(prediction_gpu)
+            assert set(prediction_cpu.keys()) == {"verbs"}
```
Comment (DeNeutoy) [inline tests/service/predictors/srl_test.py]: This test is a bit redundant - we are checking that two different serialised models produce the same thing. Don't we want to be checking that loading a gpu trained model onto the cpu gives the same predictions as a gpu model loaded onto the gpu? Maybe i've got the wrong idea here but it seems a bit lucky that this is passing, and more to do with the fact that the model fixtures aren't trained very much.

Comment (joelgrus) [inline tests/service/predictors/srl_test.py]: originally the idea was that the weights were from running the same experiment on CPU and GPU, to check that we got the same results. we've messed with these files so much that it is lucky they're passing, and possibly they won't at some point. should I just get rid of this test?

Comment (DeNeutoy) [inline tests/service/predictors/srl_test.py]: In an ideal world we would be able to test that loading models cross device provides the same result, but that's going to be difficult given we don't have GPU backed tests. Either create this test and skip it for now, or yeah, just remove it.

**Comment by joelgrus** [comment]
ok, I removed that test and also the gpu-specific fixtures that were there just for that test

**Pull request merged**